google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -267,6 +269,8 @@ public String[] getValidValues() {
   /** Name of the 'routeToLeader' connection property. */
   public static final String ROUTE_TO_LEADER_PROPERTY_NAME = "routeToLeader";
   /** Name of the 'retry aborts internally' connection property. */
+  public static final String ENABLE_END_TO_END_TRACING_PROPERTY_NAME = "enableEndToEndTracing";
```

Comment:
It now seems like the comment above belongs to this property, which it doesn't. Please also don't add any additional empty lines to this part of the file, unless it is to separate logical groups of properties that belong together.
In this case, we already have a number of tracing properties. Please add this at the same place as those (so around line 337)